R/params.R

```diff
@@ -175,6 +175,13 @@ params_configurable <- function(param) {
   length(param$value) <= 1 && !is.null(params_get_control(param))
 }
 
+# Returns a new empty named list.
+params_namedList <- function() {
+  empty <- list()
+  names(empty) <- character()
+  empty
+}
```

Comment:
Achievement unlocked: you are now such an experienced R programmer that you don't notice when using two or more coding styles in one function.